sdks/java/extensions/histogram/build.gradle

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins { id 'org.apache.beam.module' }
```

Comment:
This is a light weight util class and it does not introduce additional dependency. Just put into java core?

java/extension are usually modules that requires a dependency that is deemed optional for beam

e.g. 
- extension/avro - Avro IOs coders, etc
- extension/gcp-core - GCP libraries
- extension/protobuf , extension/arrow, etc

Reply:
+1

Reply:
Yes, it is true that we don't have any additional dependencies now, but one thing to consider is that there might be in the future(like custom coder compression libraries or vectorized operations). With that in mind, what is the best course of action in your opinion? Should we consider these possible improvements that might be implemented and leave the combiner in the extensions, or move it to the Java core?

Reply:
understand. If there is plan to extend this package, how about consider a more generic naming? Like extensions/transforms ; extensions/combiners ; extensions/stats

Reply:
like extensions/combiners/histogram? sounds good, thanks

Reply:
thanks, I mean we can have a "module" under extensions, and histogram is a package inside that "module". Each "module" will result in an release artifact. I was suggestring this and the future extension under the same module. The filesystem structure could look like this

```
- sdks/java/extensions/transforms
  |- build.gradle
  |- src/main/java/org/apache/beam/sdk/extensions/transforms/combiners/Histogram.java
  |- src/main/java/org/apache/beam/sdk/extensions/transforms/combiners/HistogramTest.java
```

This is refer to the structure of `sdks/java/core/src/main/java/org/apache/beam/sdk/transforms`, where different transforms are under same package and in different class, there isn't another layer of package (transforms/dofn/..., transforms/combine/...)

Reply:
it was just change the package naming from histogram -> combiners from 60f16728108d7fab46f8018ff9dc508df04edde7 and no other change required

Reply:
oh okay, I thought it should be in separate modules